### PR TITLE
Fix build break in outer loop runtime tests

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: src/coreclr/jit/lsra.cpp [hunk 1]
```diff
@@ -507,6 +507,46 @@ regMaskTP LinearScan::getConstrainedRegMask(RefPosition* refPosition,
     return newMask;
 }
 
+//------------------------------------------------------------------------
```
Comment (jkotas) [inline src/coreclr/jit/lsra.cpp:510]: This is fixing build break in Linux debug builds (looks like a bug in the C++ compiler/linker to me):

https://dnceng-public.visualstudio.com/public/_build/results?buildId=672464&view=logs&j=689957d3-95e5-570c-99ed-eaa47e031b3f&t=a66b4246-3d14-52ac-fc42-6b6b8bdf0ccd&l=3374

File: src/tests/ilverify/TestDataLoader.cs [hunk 2]
```diff
@@ -279,10 +279,10 @@ public TestResolver(Dictionary<string, string> simpleNameToPathMap)
                 _simpleNameToPathMap = simpleNameToPathMap;
             }
 
-            PEReader IResolver.ResolveAssembly(AssemblyName assemblyName)
+            PEReader IResolver.ResolveAssembly(AssemblyNameInfo assemblyName)
```
Comment (jkotas) [inline src/tests/ilverify/TestDataLoader.cs:282]: This is fixing build break in ILVerify tests: https://dnceng-public.visualstudio.com/public/_build/results?buildId=672464&view=logs&j=3725811e-4a8b-5f6a-b03b-a353ee894148&t=d59fccfb-2b13-5841-75bf-bfec705ee16a

**Review by tannergooding — APPROVED** [review]

File: src/tests/ilverify/TestDataLoader.cs (hunk 2 above)
Comment (kunalspathak) [reply src/tests/ilverify/TestDataLoader.cs:282]: Wondering why this was not caught in CI ran during PR. Is the difference just `Debug` vs. `Release`?

```
fail: /__w/1/s/build.sh -ci -arch arm64 -os linux -cross -s clr -c Debug 
pass: /__w/1/s/build.sh -ci -arch arm64 -os linux -cross -s clr+libs+host+packs -rc Release -c Release -lc Release   
```

Comment (jkotas) [reply src/tests/ilverify/TestDataLoader.cs:282]: Right, the default CI mix builds checked and release coreclr builds only. 99.9+% of debug build breaks are caught by checked builds. It is only outliers like this one (that looks like a C++ compiler bug to me) that can sneak through.

**Pull request merged**